packages/@aws-cdk/aws-ec2/lib/peer.ts

```diff
@@ -77,6 +77,16 @@ class CidrIPv4 implements IPeer {
   public readonly uniqueId: string;
 
   constructor(private readonly cidrIp: string) {
+    const cidrMatch = cidrIp.match(/^(\d{1,3}\.){3}\d{1,3}(\/\d+)?$/);
```

Comment:
I guess to be safe this needs a `if (!cdk.Token.isUnresolved(cidrIp)) { ... }` block around the checks, same for the IPv6 version.

Reply:
Done